src/Tizen.NUI/src/public/WebView/WebContext.cs

```diff
@@ -422,7 +422,7 @@ public bool DeleteWebStorage(WebSecurityOrigin origin)
         /// <summary>
         /// Deletes local fileSystem.
```

Comment:
whole file system? it might be good to add what does "file system" mean in this context, what's in it?

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520